src/common/Widget.js

```diff
@@ -452,7 +452,13 @@
     Widget.prototype.rowToObj = function (row) {
         var retVal = {};
         this.columns().forEach(function(col, idx) {
-            retVal[col] = row[idx];
+            var item;
+            if (row[idx] instanceof Widget) {
+                item = row[idx]._class;
```

Comment:
You can add an extra function to JSON.stringify which will let you skip Objects (this issue is probably only happening in a console.log call?).